pylti/tests/test_flask.py

```diff
@@ -454,12 +460,18 @@ def request_callback(request, cburi, headers):
         self.assertFalse(self.has_exception())
         self.assertEqual(ret.data, "grade=False")
 
+    def request_callback(self, request, cburi, headers):
```

Comment:
`register_uri` takes a "function" which accepts three parameters. This is an "instance method" which takes four parameters.